theme/palette/elements.less

```diff
@@ -7,3 +7,4 @@
 @sk-footer: @sk-mid-gray-light;
 @sk-background: @sk-gray-light;
 @sk-yellow: #fffce3;
+@sk-gold: #ffc600;
```

Comment:
That's exactly my proposition. `@sk-yellow` should be the colour defined in the brand guidelines (currently it's not), everything in `elements.less` is largely that, element colours that are not captured in the core brand colours. @markdalgleish @stephaniemoss?